documentation/modules/overview/con-configuration-points-common.adoc

```diff
@@ -14,6 +14,8 @@ Bootstrap servers:: Bootstrap servers are used for host/port connection to a Kaf
 * Kafka Bridge
 * Kafka MirrorMaker producers and consumers
 CPU and memory resources:: You request CPU and memory resources for components. Limits specify the maximum resources that can be consumed by a given container.
++
+When you define the specification for your Kafka cluster using the `Kafka` resource, you can set resource usage limits for the Topic Operator and User Operator. 
```

Comment:
Thsi sounds a bit weird. Can't we just say _Resource requests and limits for TO and UO can be set in the Kafka CR_?